crates/sncast/src/response/print.rs

```diff
@@ -87,6 +87,9 @@ impl Format for OutputValue {
                 if let Ok(field) = Felt::from_str(&input) {
                     return match numbers {
                         NumbersFormat::Decimal => OutputValue::String(format!("{field:#}")),
+                        NumbersFormat::Hex if input.len() == 66 && input.starts_with("0x0") => {
+                            OutputValue::String(input)
+                        }
```

Comment:
I guess this might not be 100% foolproof but with current printing setup it's the best one I found working.